src/tokens/tokens.interfaces.ts

```diff
@@ -15,14 +15,23 @@
 // limitations under the License.
 
 import { ApiProperty, OmitType } from '@nestjs/swagger';
-import { IsNotEmpty, IsOptional } from 'class-validator';
+import { IsDefined, IsEnum, IsNotEmpty, IsOptional } from 'class-validator';
 
 // Ethconnect interfaces
 export interface EthConnectAsyncResponse {
   sent: boolean;
   id: string;
 }
 
+export interface EthConnectContractsResponse {
```

Comment:
I think you deleted the only usage of this interface, so it can probably be removed